AlphaWallet/Protection/ViewControllers/SplashViewController.swift

```diff
@@ -8,14 +8,29 @@ class SplashViewController: UIViewController {
 
     init() {
         super.init(nibName: nil, bundle: nil)
-        splashView.translatesAutoresizingMaskIntoConstraints = false
-        view.addSubview(splashView)
-        NSLayoutConstraint.activate([
-            splashView.topAnchor.constraint(equalTo: view.topAnchor),
-            splashView.leadingAnchor.constraint(equalTo: view.leadingAnchor),
-            splashView.bottomAnchor.constraint(equalTo: view.bottomAnchor),
-            splashView.trailingAnchor.constraint(equalTo: view.trailingAnchor),
-        ])
+        splashView.backgroundColor = UIColor.red; splashView.translatesAutoresizingMaskIntoConstraints = false
+        splashView.frame =  UIScreen.main.bounds
+       view.addSubview(splashView)
+       if UIDevice.current.hasNotch {
+        if #available(iOS 11, *) {
```

Comment:
* We should always follow the existing indentation style in the file (or if that file is really bad, fix the file).
* The app doesn't support < iOS 11, so the check is not needed

Reply:
ok sure I am making note of this.